### DISCUSSION TITLE
Why does the Play Store want to update Firefox when it's the same version is already installed via Obtainium?

### DISCUSSION THREAD

**Discussion by patrickstedem** [ImranR98/Obtainium#1849] [Q&A] [answered]
I'm using Obtainium to install/update Firefox. Typically, the newest versions are available a day or so ahead of the Play Store. After I update, the Play Store shows no update is available (as I'm a version ahead) and all is well.

Once that same version becomes available in the Play Store, it shows up as an available update to be installed. Why does it say an update is available when it's the same version?

Below shows my installed Firefox version and the Play Store wanting to update to the same version.

![Screenshot_20240921-200706](https://github.com/user-attachments/assets/aee3fe7c-9885-4205-b37e-368bac54ba62)

![Screenshot_20240921-200759](https://github.com/user-attachments/assets/d529ad2f-1a9f-4857-a760-0ce7678c5450)

**Comment by ImranR98** [comment, marked as answer]
The build number is probably different. They might have some libraries/features in one that isn't in the other.

**Reply by DarkCrypt** [reply]
ImranR98 is correct, different build number (2016044751).

I highly recommend using Aurora Store instead of the Playstore. It provides much more info and you can also blacklist or "ignore" updates for Playstore apps which eliminates this issue entirely.

**Reply by DwainZwerg** [reply]
You can also deactivate auto-updates in the Play Store …

**Reply by patrickstedem** [reply]
Thanks all! I can see now that the build difference is universal vs. arm64-v8a. Took the advice and switched to Aurora and blacklisted Firefox.
